packages/cli-tool/src/add-ons/ui-framework/tailwind-css.ts

```diff
@@ -73,13 +74,22 @@ const addTailwindCssImport = (appPath: string): Promise<void> => {
     CliUx.ux.info('Update css imports in App.tsx');
 
     try {
-      const indexScssPath = `./${appPath}/src/App.tsx`;
+      const indexScssPath = `${appPath}/src/App.tsx`;
 
+      // When using Webpack
       replaceLine(
         indexScssPath,
         `import 'assets/stylesheets/application.scss';`,
         `import 'assets/stylesheets/application.css';`,
       );
+      replaceLine(indexScssPath, `import '@/dummy.scss';`, `import '@/dummy.css';`);
+
+      // When using Vite
```

Comment:
Maybe we should be doing it under if condition? I understand that it will not replace the content for the opposite case. Does this plugin have information on what we are using, vite or webpack?

Reply:
8999c21 ✅  ;-)

Reply:
I think it is a wrong commit :) And I don't see fixes for it in other commits. I don't insist on it, just raised a question to consider whether it will be better or not.

Reply:
https://github.com/nimblehq/react-templates/commit/ca7ea8c070a20186bdbc9c6894bcbc2bdba366ee ✅ 🙇